### PR TITLE
Update RSPEC for 8.54 release

### LINKED ISSUE #6763: Update RSPEC before 8.54 release

### PR DESCRIPTION
**Pull request by zsolt-kolbay-sonarsource**
Fixes #6763

### CODE REVIEW

**Review by antonioaversa — APPROVED** [review]
LGTM!

File: analyzers/rspec/cs/S4433_c#.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <p>An LDAP client authenticates to an LDAP server with a "bind request" which provides, among other, a <a
-href="https://ldapwiki.com/wiki/Simple%20Authentication">simple authentication method</a>.</p>
+href="https://web.archive.org/web/20220922153922/https://ldapwiki.com/wiki/Simple%20Authentication">simple authentication method</a>.</p>
```
Comment (pavel-mikula-sonarsource) [inline analyzers/rspec/cs/S4433_c#.html:2]: This is a broken link

Comment (pavel-mikula-sonarsource) [reply analyzers/rspec/cs/S4433_c#.html:2]: Ah, no, it works. It just takes time to Verify

File: analyzers/rspec/cs/S4433_c#.html [hunk 2]
```diff
@@ -30,6 +30,7 @@ <h2>See</h2>
   <li> <a href="https://owasp.org/www-project-top-ten/2017/A2_2017-Broken_Authentication">OWASP Top 10 2017 Category A2</a> - Broken Authentication
   </li>
   <li> <a href="https://cwe.mitre.org/data/definitions/521">MITRE, CWE-521</a> - Weak Password Requirements </li>
-  <li> <a href="https://ldapwiki.com/wiki/Simple%20Authentication">ldapwiki.com</a>- Simple Authentication </li>
+  <li> <a href="https://web.archive.org/web/20220922153922/https://ldapwiki.com/wiki/Simple%20Authentication">ldapwiki.com</a>- Simple Authentication
```
[the remark this answers is not in the document]
Comment (pavel-mikula-sonarsource) [reply analyzers/rspec/cs/S4433_c#.html:33]: same - valid

**Pull request merged**